ui/admin/tests/e2e/tests/target.spec.js

```diff
@@ -109,3 +119,38 @@ test('Verify session created then cancel the session', async ({ page }) => {
     }
   }
 });
+
+test('Verify session created to target with address then cancel the session', async ({page}) => {
```

Comment:
```suggestion
test('Verify session created to target with address, then cancel the session', async ({page}) => {
```
We may also want to update the name of the first test to something like
`Verify session created to target with host, then cancel the session`